test/v3/fixtures/retrieveBalanceAndTransactions/DocumentDetailsFixture.scala

```diff
@@ -145,13 +146,42 @@ object DocumentDetailsFixture {
        |  "effectiveDateOfPayment": "2021-04-05",
        |  "latePaymentInterest": $latePaymentInterestJson,
        |  "amountCodedOut": 5.99,
-       |  "reducedCharge": $reducedChargeJson,
-       |  "poaRelevantAmount":5.99
+       |  "reducedCharge": $reducedChargeJson
        |}
        |""".stripMargin)
       .as[JsObject]
 
+  val documentDetailsMtdResponseWithPOARelevantAmountJson: JsObject =
```

Comment:
Can `documentDetailsMtdResponseJson` be renamed to `documentDetailsWithoutPOAAmountMtdResponseJson` and this one be renamed to `documentDetailsMtdResponseJson`.